pkg/overlay/cache.go

```diff
@@ -128,3 +131,18 @@ func (o *Cache) Put(ctx context.Context, nodeID storj.NodeID, value pb.Node) err
 
 	return o.DB.Put(nodeID.Bytes(), data)
 }
+
+// Delete will remove the node from the cache. Used when a node hard disconnects or fails
+// to pass a PING multiple times.
+func (o *Cache) Delete(ctx context.Context, id storj.NodeID) error {
+	if id.IsZero() {
+		return ErrDelete
```

Comment:
Shouldn't this be `ErrEmptyNode`? Like in Get method.

Reply:
Fixed